tools/cordova/index.js

```diff
@@ -16,13 +16,13 @@ export const CORDOVA_PLATFORMS = ['ios', 'android'];
 export const CORDOVA_DEV_BUNDLE_VERSIONS = {
   'cordova-lib': '10.0.0',
   'cordova-common': '4.0.2',
-  'cordova-create': '3.0.0',
+  'cordova-create': '2.0.0',
```

Comment:
Is it a downgrade @costinelmarin? if we don't have a strong reason to downgrade, we probably should not.

Reply:
@renanccastro the android platform was not working with `'cordova-create': '3.0.0'`. I was able to make it work locally by using `2.0.0` version.

Reply:
why wasn't it working? maybe we have another solution.

Reply:
Some of the dependency were not installed, `properties-parser` package and when adding android platform I get this message `%% cordova-android version check failed (project/.meteor/local/cordova-build/platforms/android/cordova/version), continuing anyways.`.